awscli/customizations/dynamodb/subcommands.py

```diff
@@ -176,6 +173,12 @@ def _run_main(self, parsed_args, parsed_globals):
         return 0
 
     def _select(self, parsed_args, parsed_globals):
+        output_type = parsed_globals.output
+        if output_type is not None and output_type != 'yaml':
```

Comment:
Lets hoist the constant `yaml` up to a `_SUPPORTED_OUTPUT_TYPES` and do the comparison against that collection.

Reply:
Makes sense.  Just pushed up the change